docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -333,3 +333,57 @@ Mouse event | Touch event
 `mousemove` (when hovering or dragging) | `touchmove` (dragging only)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Examples
+
+### Download Files in IE
+
+TestCafe prevents native browser dialogs, including those before file download. In Internet Explorer, however, you should disable this dialog in browser settings, because it cannot be dismissed programmatically.
```

Comment:
Add the link to the Internet Explorer documentation where describes how to disable the dialog.

Reply:
added